gradoop-common/src/test/java/org/gradoop/common/model/impl/properties/PropertyValueUtilsTest.java

```diff
@@ -0,0 +1,733 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.common.model.impl.properties;
+
+import org.junit.Test;
+
+import java.math.BigDecimal;
+
+import static org.gradoop.common.GradoopTestUtils.*;
+import static org.gradoop.common.model.impl.properties.PropertyValueUtils.Boolean.or;
+import static org.gradoop.common.model.impl.properties.PropertyValueUtils.Numeric.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import static org.gradoop.common.model.impl.properties.PropertyValue.create;
+
+public class PropertyValueUtilsTest {
```

Comment:
Please pay attention to the usage of blank spaces. Although the checkstyle is not applied in the unit tests, we should still keep in mind to ensure readability etc.

> All binary operators except dot (.) should be separated from their operands by spaces.

> A blank space should appear after commas in argument lists.